lua/compe_omni/init.lua

```diff
@@ -0,0 +1,27 @@
+local compe = require("compe")
+local Source = {}
+
+function Source.new()
+  return setmetatable({}, { __index = Source })
+end
+
+function Source.get_metadata(_)
+  return {
+    priority = 100;
+    dup = 0;
+    menu = '[Omni]';
+  }
+end
+
+function Source.determine(_, context)
+  return compe.helper.determine(context)
+end
```

Comment:
I think we should check `vim.bo.omnifunc ~= ''` and use `vim.api.nvim_call_function(omnifunc, { 1, '' })` to determine `keyword_pattern_offset` instead of `compe.helper.determine`.

Reply:
The `Source.determine` function can be return empty dict (it means cancelling completion).
If `omnifunc` is empty, we should return empty on `Source.determine`.